src/components/content-tab-import-export.tsx

```diff
@@ -1,6 +1,111 @@
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { format } from 'date-fns';
+import { sanitizeFolderName } from '../lib/generate-site-name';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { ExportOptions } from '../lib/import-export/export/types';
+
 interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
-export function ContentTabImportExport( _props: ContentTabImportExportProps ) {
-	return null;
+
+const getFileName = ( selectedSite: SiteDetails ) => {
+	const timestamp = format( new Date(), 'yyyy-MM-dd-HH-mm-ss' );
+	return sanitizeFolderName( `studio-backup-${ selectedSite.name }-${ timestamp }` );
+};
+
+export const ExportSite = ( {
+	selectedSite,
+	onExport,
+}: {
+	selectedSite: SiteDetails;
+	onExport: ( options: ExportOptions ) => Promise< void >;
+} ) => {
+	const onExportFullSite = async () => {
+		const fileName = getFileName( selectedSite );
+		const path = await getIpcApi().showSaveAsDialog( {
+			title: __( 'Save backup file' ),
+			defaultPath: `${ fileName }.tar.gz`,
+			filters: [
+				{
+					name: '*.tar.gz, *.tzg *.zip',
```

Comment:
I think the name should be something informative.
```suggestion
					name: 'Compressed Backup Files',
```